src/bot/mod.rs

```diff
@@ -19,6 +21,7 @@ impl Bot {
     /// If cannot get the `TELOXIDE_TOKEN` environmental variable.
     ///
     /// [`reqwest::Client`]: https://docs.rs/reqwest/0.10.1/reqwest/struct.Client.html
+    #[allow(deprecated)]
```

Comment:
`#[depredecated]`?

Reply:
I think that this function is anyway frequently used in the examples, so we don't need to deprecate it.